staging/src/k8s.io/client-go/applyconfigurations/core/v1/pod.go

```diff
@@ -0,0 +1,173 @@
+	return b
+}
+
+// SetUID sets the UID field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetUID(value types.UID) *PodApplyConfiguration {
+	b.UID = &value
+	return b
+}
+
+// SetResourceVersion sets the ResourceVersion field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetResourceVersion(value string) *PodApplyConfiguration {
+	b.ResourceVersion = &value
+	return b
+}
+
+// SetGeneration sets the Generation field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetGeneration(value int64) *PodApplyConfiguration {
+	b.Generation = &value
+	return b
+}
+
+// SetCreationTimestamp sets the CreationTimestamp field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetCreationTimestamp(value metav1.Time) *PodApplyConfiguration {
+	b.CreationTimestamp = &value
+	return b
+}
+
+// SetDeletionTimestamp sets the DeletionTimestamp field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetDeletionTimestamp(value metav1.Time) *PodApplyConfiguration {
+	b.DeletionTimestamp = &value
+	return b
+}
+
+// SetDeletionGracePeriodSeconds sets the DeletionGracePeriodSeconds field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetDeletionGracePeriodSeconds(value int64) *PodApplyConfiguration {
+	b.DeletionGracePeriodSeconds = &value
+	return b
+}
+
+// SetLabels sets the Labels field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetLabels(value map[string]string) *PodApplyConfiguration {
+	b.Labels = value
+	return b
+}
+
+// SetAnnotations sets the Annotations field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetAnnotations(value map[string]string) *PodApplyConfiguration {
+	b.Annotations = value
+	return b
+}
+
+// SetOwnerReferences sets the OwnerReferences field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetOwnerReferences(values ...*v1.OwnerReferenceApplyConfiguration) *PodApplyConfiguration {
+	s := make([]v1.OwnerReferenceApplyConfiguration, len(values))
+	for i := range values {
+		s[i] = *values[i]
+	}
+	b.OwnerReferences = s
+	return b
+}
+
+// SetFinalizers sets the Finalizers field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetFinalizers(value []string) *PodApplyConfiguration {
+	b.Finalizers = value
+	return b
+}
+
+// SetClusterName sets the ClusterName field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetClusterName(value string) *PodApplyConfiguration {
+	b.ClusterName = &value
+	return b
+}
+
+// SetManagedFields sets the ManagedFields field in the declarative configuration to the given value.
+func (b *PodApplyConfiguration) SetManagedFields(values ...*v1.ManagedFieldsEntryApplyConfiguration) *PodApplyConfiguration {
```

Comment:
This is probably a hard-code, but this one we really know it shouldn't be sent :-)

Reply:
Good catch! I'll block it with a hard-coded rule.